docs/howto/custom-management-commands.txt

```diff
@@ -8,7 +8,7 @@ Applications can register their own actions with ``manage.py``. For example,
 you might want to add a ``manage.py`` action for a Django app that you're
 distributing. In this document, we will be building a custom ``closepoll``
 command for the ``polls`` application from the
-:doc:`tutorial</intro/tutorial01>`.
+:doc:`tutorial</intro/tutorial02>`.
```

Comment:
I think we can leave here `tutorial01` because it starts `Creating the Polls app`.